gwctl/pkg/printer/gateways.go

```diff
@@ -24,12 +24,13 @@ import (
 	"strings"
 	"text/tabwriter"
 
+	"sigs.k8s.io/yaml"
```

Comment:
nit: Are these diffs (including httproutes.go, policies.go) a result of some linter complaining? If not, can we minimize such changes.